packages/jest-emotion/src/index.js

```diff
@@ -56,43 +56,76 @@ type Options = {
   DOMElements?: boolean
 }
 
-function filterEmotionProps(props = {}) {
+function filterEmotionProps(props = {}, shouldKnowStyles = true) {
   const {
     css,
     __EMOTION_TYPE_PLEASE_DO_NOT_USE__,
     __EMOTION_LABEL_PLEASE_DO_NOT_USE__,
     ...rest
   } = props
 
-  rest.css = 'unknown styles'
+  if (shouldKnowStyles) {
```

Comment:
Could you explain how this works now? Looks a little bit quirky - why this gets set here based on the argument and without any condition based on actual `props`?

Reply:
This was really an effort to keep existing snapshots the same, but omit "unknown styles" in the shallow rendering. On reflection, it's probably best to simply remove this condition.